lib/src/io_file.dart

```diff
@@ -262,6 +344,22 @@ class _FromXFile implements IOFile {
   Future<String> readAsString() {
     return _file.readAsString();
   }
+    
+  @override
+  Stream<Uint8List> openReadStream([int start = 0, int? end]) async*{
+    int globalOffset = start;
+    int globalEnd = end ?? await _file.length();
+    while (globalOffset < globalEnd) {
+      final chunkEnd = globalOffset + readStreamChunkSize > globalEnd
+          ? globalEnd
+          : globalOffset + readStreamChunkSize;
+
+      final chunk = await collectBytes(_file.openRead(globalOffset, chunkEnd));
+      yield chunk;
+
+      globalOffset += readStreamChunkSize;
+    }
+  }
```

Comment:
Any reason why we needed to override this? - I've already seen this code here -> https://github.com/ar-io/flutter_file_picker/pull/2 🤔

Reply:
This method is from `IOFile`. The other method is from the file picker